compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTileAndDistribute.cpp

```diff
@@ -65,8 +65,38 @@ static void populateTilingReductionPatterns(RewritePatternSet &patterns) {
           StringAttr::get(context, getWorkgroupMemoryMarker())},
       StringAttr::get(context, getWorkgroupKTiledMarker()));
   filter.setMatchByDefault();
-  TilingPatterns<linalg::MatmulOp, linalg::BatchMatmulOp,
-                 linalg::GenericOp>::insert(patterns, tilingOptions, filter);
+  TilingPatterns<linalg::MatmulOp, linalg::BatchMatmulOp, linalg::GenericOp,
+                 linalg::Conv2DNhwcHwcfOp,
+                 linalg::Conv2DNchwFchwOp>::insert(patterns, tilingOptions,
+                                                   filter);
+}
+
+LogicalResult tileToSerialLoops(func::FuncOp funcOp) {
```

Comment:
nit: This looks like the only remaining reference to `tileToSerialLoops` and can be marked static.

Reply:
Done